crates/rolldown/src/types/tree_shake.rs

```diff
@@ -5,9 +5,8 @@ bitflags::bitflags! {
   #[derive(Default, Copy, Clone, Debug)]
   pub struct UsedInfo: u8 {
     // If the `NormalModule#namespace_object_ref` is used
-    const USED_AS_NAMESPACE = 1 << 1;
-    // TODO(hyf0): suspicious
-    const INCLUDED_AS_NAMESPACE = 1 << 2;
```

Comment:
recursive including all export is sometimes expensive, this flag is just used for marking if a module is recursively included.